superset/datasets/api.py

```diff
@@ -280,6 +284,11 @@ def put(self, pk: int) -> Response:
             500:
               $ref: '#/components/responses/500'
         """
+        override_column = (
+            request.args["override_column"]
+            if request.args.get("override_column")
+            else False
+        )
```

Comment:
You can write this as:

```python
override_column = request.args.get("override_column", False)
```

But I think this is not what we want, because when we pass `override_column=false` we would get the string `"false"`, which is a truthy value. You need to convert the string to a bool:

```python
>>> from distutils.util import strtobool
>>> strtobool("false")
0
>>> strtobool("true")
1
```

So we probably want this:

```python
override_column = (
    bool(strtobool(request.args["override_column"]))
    if "override_column" in request.args else False
)
```

Can be also standardize and call this parameter `override_columns`, plural? Right now we have singular coming from the browser and plural in the backend.